flutter/Graminsta/lib/createPostWidget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+
+class CreatePostWidget extends StatefulWidget {
+  CreatePostWidget({Key key}) : super(key: key);
+
+  @override
+  _CreatePostWidgetState createState() => _CreatePostWidgetState();
+}
+
+class _CreatePostWidgetState extends State<CreatePostWidget> {
+  String accessSetting = "All";
```

Comment:
I'd suggest to use a enum instead.

```
enum AccessControlType {
  public,
  private,
  visibleToSpecificFriends,
  invisibleToSpecificFriends,
}
```

and maybe a helper function to return `String` based on the enum.

```
extention AccessControlTypeExtension on AccessControlType {
  String toHumanReadableString() {
    switch(this):
      case AccessControlType.public:
        return "ALL";
  ...
}
```

See also: https://stackoverflow.com/questions/38908285/add-methods-or-values-to-enum-in-dart